docs/readme.ptxt

```diff
@@ -406,7 +407,17 @@ their work directly to OpenSFX:
 * Timo A. Hummel ("Felicitus" at "dev.openttdcoop.org")
     * Own work
 
-4.4) License reference
+4.4) Translators
+==== ===========
+* jfs: Danish
+* Erato: Dutch, Esperanto
+* ALEXbr: Romanian
+* Juanjo: Catalan
+* silpol: Russian, Ukrainian, Polish, Finnish
+* Chrill: Swedish
+* Wuzzy: German
+
```

Comment:
We don't have this in any of our other projects, no need to add it here. Attribution in the commit message like eints commits would seems appropriate

Also, be sure to use correct commit style in your commit messages (you'll need to rebase to reword them)